titiler/endpoints/cog.py

```diff
@@ -6,6 +6,7 @@
 from urllib.parse import urlencode
 
 from rasterio.transform import from_bounds
+from rio_cogeo.cogeo import cog_validate
```

Comment:
@geospatial-jeff what do you think about using `cog_info` here?

Reply:
Makes sense to me with new rio-cogeo release

Reply:
the goal is to be able to print info in addition to `is_valid` (and not use /cog/info) 

I'll work on a small web page after this PR ;-)